test/runtime/runtime.shared.js

```diff
@@ -0,0 +1,271 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { clear_loops, flush, set_now, set_raf } from 'svelte/internal';
+import { setImmediate } from 'timers/promises';
+import glob from 'tiny-glob/sync.js';
+import { afterAll, assert, beforeAll, describe, it } from 'vitest';
+import { compile } from 'svelte/compiler';
+import { create_loader, mkdirp, try_load_config } from '../helpers.js';
+import { setup_html_equal } from '../html_equal.js';
+
+let unhandled_rejection = false;
+function unhandledRejection_handler(err) {
```

Comment:
this casing is a bit funny to me. I probably would have gone with:
```suggestion
function unhandled_rejection_handler(err) {
```